modules/core/src/main/java/org/apache/ignite/IgniteSystemProperties.java

```diff
@@ -53,30 +149,40 @@
      * @deprecated Use {@link #IGNITE_LOG_INSTANCE_NAME}.
      */
     @Deprecated
+    @SystemProperty(value = "If this system property is present Ignite will include grid name into " +
+        "verbose log", type = String.class)
     public static final String IGNITE_LOG_GRID_NAME = "IGNITE_LOG_GRID_NAME";
 
     /**
      * If this system property is present the Ignite will include instance name into verbose log.
      */
+    @SystemProperty(value = "If this system property is present Ignite will include instance name " +
```

Comment:
`SKIP_MAP_CHECK` is missed in `IgniteSystemProperties` but exists in the source code.